libs/contrib/Data/String/BuildExpressionParser.idr

```diff
@@ -0,0 +1,132 @@
+-- Port of https://hackage.haskell.org/package/parsec-3.1.13.0/docs/Text-Parsec-Expr.html
+-- Original License	BSD-style: https://hackage.haskell.org/package/parsec-3.1.13.0/src/LICENSE
+
+module Data.String.BuildExpressionParser
```

Comment:
Maybe it could be called `Data.String.Parser.Expression`?

Reply:
Can do! Would I need to nest the file in a `parser/` parent dir too, or can I just rename it to `Expression.idr`? Still getting the hang of modules...

Reply:
The module path is the directory path, so you'd put it in a `Parser` dir.

Reply:
Updated.